packages/graphql-toolbox/src/components/editor/utils.ts

```diff
@@ -69,3 +69,14 @@ export const handleEditorDisableState = (mirror: EditorFromTextArea | null, load
         wrapperElement?.classList.remove("code-mirror-disabled-state");
     }
 };
+
+export const isJsonString = (str: string | null | undefined): boolean => {
+    if (!str) return false;
+
+    try {
+        JSON.parse(str);
+    } catch (e) {
+        return false;
+    }
+    return true;
+};
```

Comment:
**suggestion** :thinking: As I see, you use `.parse` to check if a variable is a json, and the run `json` again [here](https://github.com/neo4j/graphql/pull/1282/files#diff-475760ba8faf849a5b1fdcca13f67834d707b37914a49f593e7fd397a9dfd0e5R75)

Obviously, performance is not a huge concern, but maybe we can avoid parsing so many times by changing this into a custom parser function, something like:

```suggestion
export const safeParse = (str: string | null | undefined, default: Record<string, any>): Record<string,any> => {
    if (!str) return default;

    try {
        return JSON.parse(str);
    } catch (e) {
        return default;
    }
};
```